### PR TITLE
feat(manager-components): rename to `manager-react-components`

### PR DESCRIPTION
**Pull request by aboungnaseng-ovhcloud**
| Question         | Answer
| ---------------- | ---
| Branch?          | develop
| Bug fix?         | no
| New feature?     | no
| Breaking change? | yes
| Tickets          | MANAGER-14822
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [x] Breaking change is mentioned in relevant commits

## Description

Update the package name of the manager components lib

### CODE REVIEW

File: packages/manager-components/CHANGELOG.md
Comment (anooparveti) [inline packages/manager-components/CHANGELOG.md:1]: **nitpick:** I dont think we need to update the package name in the changelog.

Comment (aboungnaseng-ovhcloud) [reply packages/manager-components/CHANGELOG.md:1]: 👍

File: .github/workflows/run-bdd-tests.yml
Comment (antleblanc) [inline .github/workflows/run-bdd-tests.yml:1]: suggestion: you can isolate such changes on a different commit with type `ci`.

**Review by chipp972 — APPROVED** [review]

File: .github/workflows/run-bdd-tests.yml
Comment (aboungnaseng-ovhcloud) [reply .github/workflows/run-bdd-tests.yml:1]: 👍

**Review by anooparveti — DISMISSED** [review]

File: .github/workflows/run-bdd-tests.yml
Comment (antleblanc) [reply .github/workflows/run-bdd-tests.yml:1]: The commit 3675b013f554db2a26ec759c4b3c05ca5fed4561 is empty.

**Review by chipp972 — APPROVED** [review]

File: .github/workflows/run-bdd-tests.yml
Comment (aboungnaseng-ovhcloud) [reply .github/workflows/run-bdd-tests.yml:1]: It should be ok now, I didnt push last change

**Review by anooparveti — APPROVED** [review]

**Review by chipp972 — APPROVED** [review]

**Pull request merged**